File-Formats/Presentation/Convert-PowerPoint-Presentation-to-Image-in-Azure-App-Service-Linux.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Convert PPTX to Image in Azure App Service on Linux | Syncfusion
+description: Convert PPTX to image in Azure App Service on Linux using PowerPoint library (Presentation) without Microsoft PowerPoint or interop dependencies.
+platform: file-formats
+control: PowerPoint
+documentation: UG
+---
+
+# Convert PowerPoint Presentation to Image in Azure App Service on Linux
+
+Syncfusion PowerPoint is a [.NET Core PowerPoint library](https://www.syncfusion.com/document-processing/powerpoint-framework/net-core) used to create, read, edit and **convert PowerPoint documents** programmatically without **Microsoft PowerPoint** or interop dependencies. Using this library, you can **convert a PowerPoint Presentation to image**.
```

Comment:
convert a PowerPoint Presentation to PDF in Azure App service on Linux

Complete the sentence in all md files.

Reply:
updated